.github/workflows/wave.yml

```diff
@@ -52,12 +52,12 @@ jobs:
     # if: github.repository == 'nf-core/modules'
     if: ${{ needs.generate-matrix.outputs.conda-matrix != '[]' }}
     needs: generate-matrix
-    name: Build ${{ matrix.profile }} ${{ matrix.platform }} Container
+    name: 🌊 ${{ matrix.files }} ${{ matrix.profile }} ${{ matrix.platform }}
     runs-on: ubuntu-latest
     timeout-minutes: 60
     strategy:
       fail-fast: false
-      max-parallel: 4
+      # max-parallel: 2
```

Comment:
why not 4? we have cores, so why not use them?

Reply:
1. I turned it off for the purpose of running them all quickly
2. I don't know exactly how that will translate to AWS ec2 runners. A lot of these jobs will just ping wave and be done. So I didn't want to spin up 20 runners because it's getting ready for a lot of runs and we're going to need a lot of capacity, and then it's 20 of the same wave runs because it was GATK.